Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Gradient.kt

```diff
@@ -45,20 +47,37 @@ fun Modifier.offsetGradientBackground(
     colors: List<Color>,
     width: Float,
     offset: Float = 0f
-) = background(
+) = this then background(
```

Comment:
I think the `this then` here is double adding the modifier because you have implicitly captured `this` already by not doing `Modifier.background` but just `background` which is actually `this.background`

We probably need another lint check for this as well!

Reply:
I'm actually not sure when you're supposed to use which. Happy to write the lint check!

Reply:
You basically just have to use `this` somewhere in your implementation. By just writing `background` you've done it